docs/learn/documentation/versioned/api/samza-sql.md

```diff
@@ -19,65 +19,57 @@ title: Samza SQL
    limitations under the License.
 -->
 
+### Table Of Contents
+- [Introduction](#introduction)
+- [Code Examples](#code-examples)
+- [Key Concepts](#key-concepts)
+  - [SQL Representation](#sql-representation)
+  - [SQL Grammar](#sql-grammar)
+  - [UDFs](#UDFs)
+  - [UDF Polymorphism](#udf-polymorphism)
+- [Known Limitations](#Known-Limitations)
 
-### Overview
-Samza SQL allows you to define your stream processing logic declaratively as a a SQL query.
-This allows you to create streaming pipelines without Java code or configuration unless you require user-defined functions (UDFs). Support for SQL internally uses Apache Calcite, which provides SQL parsing and query planning. The query is automatically translated to Samza's high level API and runs on Samza's execution engine.
+### Introduction
+Samza SQL allows you to define your stream processing logic 
+declaratively as a a SQL query. This allows you to create streaming 
+pipelines without Java code or configuration unless you require 
+user-defined functions (UDFs). 
 
-You can run Samza SQL locally on your machine or on a YARN cluster.
+### Code Examples
 
-### Running Samza SQL on your local machine
-The [Samza SQL console](https://samza.apache.org/learn/tutorials/0.14/samza-tools.html) allows you to experiment with Samza SQL locally on your machine. 
+The [Hello Samza](https://github.com/apache/samza-hello-samza/tree/master/src/main/java/samza/examples/sql) 
+SQL examples demonstrate how to use Samza SQL API.  
 
-#### Setup Kafka
-Follow the instructions from the [Kafka quickstart](http://kafka.apache.org/quickstart) to start the zookeeper and Kafka server.
+- The [sampling](https://github.com/apache/samza-hello-samza/tree/master/src/main/java/samza/examples/sql/samza-sql-samples-yarn)
```

Comment:
nit: link appears broken.. Do you have a pending PR for the SQL examples on YARN? 

if so, maybe worth getting that committed at first?

Reply:
indeed & agreed. The examples PR is https://github.com/apache/samza-hello-samza/pull/46

Reply:
This link is broken as you updated the examples. Could you fix it ?

Reply:
fixed